### PR TITLE
multiplayer moo patch

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

**someone please tell me if the time complexity is correct**

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
btw i'll add comments later

**Review by envyaims — APPROVED** [review]
nice!!!!

**Comment by SansPapyrus683** [comment]
will also close #3102 btw

**Review by ryanchou-dev — APPROVED** [review]
ily

File: solutions/usaco-836.mdx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+						curr_vis[nr] = true;
+						frontier.push_back(nr);
+					}
+				}
+			}
+			two_biggest = std::max(two_biggest, two_size);
+		}
+	}
+
+	std::ofstream("multimoo.out") << one_biggest << '\n' << two_biggest << endl;
```
Comment (ChuangSheep) [inline solutions/usaco-836.mdx:145]:
```suggestion
	std::ofstream("multimoo.out") << one_biggest << endl << two_biggest << endl;
```
maybe just use one style?

**Review by ChuangSheep — APPROVED** [review]
nice ❤️

File: solutions/usaco-836.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/usaco-836.mdx:145]: think it only makes sense to flush the output @ the end

File: solutions/usaco-836.mdx [hunk 2]
```diff
@@ -0,0 +1,150 @@
+---
+id: usaco-836
+source: USACO Silver 2018 US Open
+title: Multiplayer Moo
+author: Kevin Sheng
+---
+
+[Official Editorial (C++)](http://www.usaco.org/current/data/sol_multimoo_silver_open18.html)
+
+The solution uses an iterative approach to floodfill instead of a recursive approach.
```
[the remark this answers is not in the document]
Comment (SansPapyrus683) [reply solutions/usaco-836.mdx:10]: i mean it's in the module, no?

[the remark this answers is not in the document]
Comment (ChuangSheep) [reply solutions/usaco-836.mdx:10]: yo then it's alright

File: solutions/usaco-836.mdx (hunk 1 above)
Comment (ChuangSheep) [reply solutions/usaco-836.mdx:146]: ok... it's fine then i guesss

**Pull request merged**